src/pytsql/grammar/helper_generate_parsers.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+
+antlr4="java -jar /usr/local/lib/antlr4-4.9.2_1-complete.jar"
+
+# Generate C++ target with visitor
+$antlr4 -Dlanguage=Cpp -visitor -no-listener -o cpp_src tsql.g4
+
+# Generate Python target
+$antlr4 -Dlanguage=Python3 -no-visitor -no-listener -o . tsql.g4
+
+# Run speedy-antlr-tool to generate parse accelerator
+python3 <<EOF
+from speedy_antlr_tool import generate
```

Comment:
Nothing installs `speedy_antlr_tool` right now; I would either:
1. Add it to the instructions
2. Add it as a [`pip` dependency](https://stackoverflow.com/a/35245610) in `environment.yml`
3. (as you suggested, and I would encourage that) Add a conda feedstock for it and specify it as a `conda` dependency 🏆

Reply:
Let's try to lift the version of the antlr runtime to 4.9.3 and also regenerating the grammars